guides/plugins/apps/administration/add-cms-element-via-admin-sdk.md

```diff
@@ -9,35 +9,36 @@ nav:
 
 ## Overview
 
-This article will teach you how to create a new CMS element via the Meteor Admin SDK. The plugin in this example will be named `SwagBasicAppCmsElementExample`, similar to the other guides.
+This guide explains how to create a new CMS element using the Meteor Admin SDK. The example plugin is named
+`SwagBasicAppCmsElementExample`, following the naming conventions used in other guides.
 
 ## Prerequisites
 
-* Knowledge on the creation of [Plugins](../plugins/plugin-base-guide) or [Apps](../app-base-guide)
-* Knowledge on the [creation of custom admin components](../../plugins/administration/module-component-management/add-custom-component#creating-a-custom-component)
-* Understanding the [Meteor Admin SDK](/resources/admin-extension-sdk/getting-started/installation)
+* Familiarity with creating [Plugins][plugins-guide] or [Apps][apps-guide].
+* Familiarity with [creating custom admin components][custom-component-creation]
+* Understanding of the [Meteor Admin SDK][meteor-sdk-installation]
```

Comment:
```suggestion
* Understanding the [Meteor Admin SDK][meteor-sdk-installation]
```
I think since these are bullet points, it was better without the "of"